codec_specs.md

```diff
@@ -14,6 +14,46 @@ Important Note:
 
 The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (cellar at ietf.org).
 
+## Defining Matroska Codec Support
+
+Support for a codec is defined in Matroska with the following values.
+
+### Codec ID
+
+Each codec supported for storage in Matroska MUST have a unique Codec ID. Each Codec ID MUST be prefixed with a two character string according to the associated type of the codec as listed in the following table.
+
+Codec Type | Codec ID Prefix
+-----------|----------------
+Video      | V_
+Audio      | A_
+Subtitle   | S_
+Button     | B_
+
+Each Codec ID MUST then use a `Major Codec ID` and may optionally use a `Minor Codec ID` and `Micro Codec ID` with a forward-slash (`/`) delimiter between each component.
```

Comment:
In some versioning systems, the fourth value is called `qualifier`. I suppose this term could fit here.

Also from codec_specs.md and https://matroska.org/technical/specs/codecid/index.html it doesn't seem possible to `know` that these Codec IDs are deprecated or what they are superseded by. Are these only discontinued by mkvmerge or by the Matroska format?

Should we have two other descriptors:
- Deprecation Date: optional but gives a date when the Matroska specification maintain deprecates a particular Codec ID.
- Superseded By: if Deprecation Date is used, then it should be highly recommended to say what Codec ID has superseded the deprecated one.

For players, they could then use the full registry for playback support, though writers should avoid using deprecated Codec IDs.